optimum/exporters/onnx/model_configs.py

```diff
@@ -701,6 +702,9 @@ def outputs(self) -> Mapping[str, Mapping[int, str]]:
 
     @property
     def values_override(self) -> Optional[Mapping[str, Any]]:
+        # TODO: do we always need to use cache?
```

Comment:
@mht-sharma question for you

Reply:
Yes, we always want the model to output the `past_key_values`. This is because the `ORTModelForSeq2SeqLM` and derivatives uses two decoder models for inference (with past and without past).

Reply:
Alright, so keeping it like that, thanks for the explaination!